src/routes/+error.svelte

```diff
@@ -12,37 +12,34 @@
 </svelte:head>
 
 <section title={$page.error?.message}>
-  <em>404</em>
-  <h1>Frank can't find where you're going!</h1>
-  <Button text={'Return to Home'} link={'/'} />
-  <img src="/assets/png/lost-frank.png" alt="404 - Page Not Found" />
+  <div class="container">
+    <div>
+      <em>404</em>
+      <h1>Frank can't find where you're going!</h1>
+      <Button text={'Return to Home'} link={'/'} />
+    </div>
+    <img src="/assets/png/lost-frank.png" alt="404 - Page Not Found" />
+  </div>
 </section>
 
 <Footer />
 
 <style>
-  section {
+  .container {
     min-height: 100vh;
     display: flex;
     gap: 2rem;
+    display: flex;
     flex-direction: column;
     justify-content: center;
     align-items: center;
-    margin: 0 24px;
-    color: var(--acm-light);
-  }
-
-  section h1 {
-    align-items: center;
+    height: 120vh;
```

Comment:
I thought that giving the section a height of >100vh would do the trick. It is close, but not exactly what we need.

What we could do is simply add back your technique from earlier by setting padding to the top of the container to offset the space left under the navbar.

With this offset `padding-top: 100px`, we may remove `height: 120vh` _in favor of the existing `min-height: 100vh`_.

```suggestion
    padding-top: 100px;
```

Reply:
Resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/815/commits/af5cc8fd621356f807183cc4db231fe192b041d4>.